plugins/kapt3/kapt3-base/test/org/jetbrains/kotlin/kapt3/base/incremental/JavaClassCacheManagerTest.kt

```diff
@@ -155,38 +157,14 @@ class JavaClassCacheManagerTest {
 
         val dirtyFiles =
             cache.invalidateAndGetDirtyFiles(
-                listOf(File("Constants.java")), emptyList()
+                listOf(File("Constants.java").absoluteFile), emptyList()
             ) as SourcesToReprocess.Incremental
         assertEquals(
             listOf(File("Constants.java").absoluteFile, File("MentionsConst.java").absoluteFile),
             dirtyFiles.toReprocess
         )
     }
 
-    @Test
-    fun testWithAnnotations() {
```

Comment:
It's not clear why this test was deleted

Reply:
`invalidateEntriesAnnotatedWith` function is removed and we are now relying on `IncrementalAptCache.aggregatedTypes` to find all types annotated by aggregating annoattions.